app/components/NavigationButtom/index.js

```diff
@@ -1,2 +1,52 @@
-import { NavigationButton } from './NavigationButton';
-export default NavigationButton;
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from 'material-ui/styles';
+import BottomNavigation, {
+  BottomNavigationAction,
+} from 'material-ui/BottomNavigation';
+import Button from 'material-ui/Button';
+// ICONS
+import HumanGreeting from 'mdi-material-ui/HumanGreeting';
+import Silverware from 'mdi-material-ui/Silverware';
+import CurrencyUsd from 'mdi-material-ui/CurrencyUsd';
+
+import { styles, BottomWrapper, ButtonContainer } from './styles';
+
+class SimpleBottomNavigation extends Component {
+  state = {
+    value: 0,
```

Comment:
can you add this to the store? we may want to use this information across the app.

Reply:
you can use the `table` reducer and just add a `namespace` called something like `section` just so that the whole app will now the current screen.
so section can be either `menu` or `bills` or something along this lines, makes sense?

Reply:
I will fix this